docs/third_party.rst

```diff
@@ -299,3 +299,6 @@ ask to raise the status.
 
 - `nacl_middleware <https://github.com/CosmicDNA/nacl_middleware>`_
   An aiohttp middleware library for asymmetric encryption of data transmitted via http and/or websocket connections.
+
+- `aiohttp-asgi-connector <https://github.com/thearchitector/aiohttp-asgi-connector>`_
+  An aiohttp connector for directly interfacing with ASGI applications.
```

Comment:
I think the wording could make it clearer that this only affects the client.
i.e. It wouldn't resolve #8233, which is asking for full integration with the server-side (replacing aiohttp's internal server).

Reply:
i clarified that it's for use with `ClientSession`